src/sections/Community/Handbook/Handbook.style.js

```diff
@@ -38,21 +38,21 @@ export const HandbookWrapper = styled.div`
 
     .page-section{
       h2{
-        padding-top: 7rem;
-        margin-top: -7rem;
+        padding-top: 0rem;
+        margin-top: 0rem;
       }
       h3{
         padding-top: 7rem;
         margin-top: -7rem;
       }
       @media (min-width: 750px) {
-        margin-top: -36rem;
+        margin-top: -53rem;
         margin-left: 20rem;
       }
       display: flex;
 
       @media screen and (min-width: 768px) and (max-height: 1145px) {
-       margin-top : -62vh ;
+       margin-top : -90vh ;
```

Comment:
This doesn't looks good @gourav-k-shaw . It means we are just setting margins as required. In future something around this changes again we have to come and set the margin. Please use some other generic css so and use as less media queries you can. So we don't have to rely on changes around this component.